Plate_1_data/1.wave2_data/README.md

```diff
@@ -0,0 +1,32 @@
+# ASC Bleedthrough Assessment - Wave 2
+
+In this wave of Plate 1 data, we have images from two different conditions with four channels, totalling 4,608 images.
+
+The conditions are as follows:
+
+For **condition 7**:
+- HOECHST 33342 -> Nuclei
```

Comment:
May consider adding a verbal description about how this mapping should be interpreted.

Reply:
I am not 100% sure what you mean here by mapping. In this section, it is going over the stains and what is being stained for in each condition.